src/ImageService.Models.Db/DbImagesMessage.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+
+namespace LT.DigitalOffice.ImageService.Models.Db
+{
+    public class DbImagesMessage
+	{
+        public const string TableName = "ImagesMessage";
```

Comment:
the table name must be plural. Add "s" to all tables in the end of the word

Reply:
done